TUM Campus App/PlansTableViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  PlansTableViewController.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 16.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class PlansTableViewController: UITableViewController {
+    
+    // MARK: - Table view data source
+    lazy var plans = Plans()
+    let loadingView = UIView()
+    let spinner = UIActivityIndicatorView()
+    let loadingLabel = UILabel()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.downloadData()
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return plans.getPlans().count
+    }
+
+    // load data into the rows
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let plan = plans.getPlan(forIndex: indexPath.row)
+        
+        let cell = tableView.dequeueReusableCell(withIdentifier: "planCell", for: indexPath) as! PlansTableViewCell
+
+        // Configure the cell...
+        cell.planTitleLabel.text = plan.title
```

Comment:
I searched for tutorials on this, but every tutorial I found, configures the cell in the VC:

- https://www.ralfebert.de/tutorials/ios-swift-uitableviewcontroller/custom-cells/
- http://www.codingexplorer.com/getting-started-uitableview-swift/#crayon-5887c31500f88513863969
- http://shrikar.com/uitableview-and-uitableviewcell-customization-in-swift/